coprocess/python/tyk/middleware.py

```diff
@@ -23,7 +23,11 @@ def __init__(self, filepath, bundle_root_path=None):
         self.imported_modules = []
         
         module_splits = filepath.split('_')
-        self.api_id, self.middleware_id = module_splits[0], module_splits[1]
+        self.api_id = ""
+        if len(module_splits) == 1:
+            self.middleware_id = module_splits[0]
+        else:
+            self.api_id, self.middleware_id = module_splits[0], module_splits[1]
```

Comment:
do we need this block? we remove `api_id` prefix from bundle destpath [here](https://github.com/TykTechnologies/tyk/pull/3849/files#diff-72df0cbfd3765a5d0bff62196c11008596608a21a53dbb9c65bfc6f008dbfa29R322). so will there be cases where the filepath having `_` in it?

Reply:
Probably not required anymore, this is still WIP, need to run a few tests

Reply:
Removed this part